test/studies/shootout/fasta/kbrady/fasta-enum.chpl

```diff
@@ -13,7 +13,7 @@ config const LOOKUP_SCALE : real = LOOKUP_SIZE - 1;
 config const n = 1000;
 
 const stdout = openfd(1).writer(kind=iokind.native, locking=false);
-param newLine: int(8) = ascii("\n");
+param newLine = ascii("\n");
```

Comment:
Even though there is no extraneous cast in this version of fasta, I think all the occurrences of int(8) should become uint(8) because they are dealing with characters.  This line is fine but look for int(8) everywhere.